src/Parallel/DistributedObject.hpp

```diff
@@ -531,6 +493,11 @@ DistributedObject<ParallelComponent, tmpl::list<PhaseDepActionListsPack...>>::
             db::wrap_tags_in<Tags::FromGlobalCache, all_cache_tags>>>(
         metavariables{}, global_cache_proxy_,
         std::move(get<InitializationTags>(initialization_items))...);
+    box_ = db::create_from<
```

Comment:
maybe for a future PR, but can we combine the `create_from` with the `create`? I don't remember what the most expensive part of DataBox construction is, but I thought `create_from` wasn't cheap at least

Reply:
Ah, yea that's more difficult. Okay, definitely a different PR if we decide to do it at all :)